### DISCUSSION TITLE
Welcome to waku Discussions!

### DISCUSSION THREAD

**Discussion by dai-shi** [wakujs/waku#149] [Announcements]
# 🤩

**Comment by dante01yoon** [comment]
🎉

**Comment by hanayashiki** [comment]
waku waku~!

**Comment by johnsonav1992** [comment]
Hey @dai-shi ! What's the name of that code color theme you used for the code snippets on the Waku website? Is that a theme I can download for VS code?

**Reply by dai-shi** [reply]
@sandren

**Reply by sandren** [reply]
It's called [Lucy](https://marketplace.visualstudio.com/items?itemName=Hermitter.oh-lucy-vscode). 🙂

**Reply by johnsonav1992** [reply]
@dai-shi @sandren Thank you so much!
